### PR TITLE
Epicurean306.github.io:journal-week-3

### PR DESCRIPTION
**Pull request by Epicurean306**
WIP branch in Epicurean306.github.io for journal-week-3

### CODE REVIEW

**Comment by pcreglow** [comment]
Since I was not able to attend the conference on Thursday, I am interested in learning more about T-shaped employees. Not sure, but an examination of that topic alone could be enough for a week 3 journal entry.

**Comment by gatorpazz** [comment]
Nice tutorial, very pertaining to the homework, good start on the outline. :+1:

File: about/html.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#Hypertext Markup Language
+* * *
+###basic TAGS
+* `<!DOCTYPE html>` — obvious function and no closing-tag
```
Comment (al-the-x) [inline about/html.md]: I wonder who finds this function obvious... Maybe a little research?

**Comment by pcreglow** [comment]
Good recap of the conference. I missed the "T" employees session so that was especially interesting. I would recommend adding a link or two to provide readers with a reference point for Chirs Coyier, SVG, etc. :+1:

**Pull request merged**